test/evergreen.yml

```diff
@@ -5198,6 +5227,7 @@ buildvariants:
     - name: ".stress-test-no-barrier-nonstandalone"
     - name: format-abort-recovery-stress-test-nonstandalone
     - name: format-predictable-test
+    - name: format-tiered-test
```

Comment:
Please make sure you have plenty of test runs of format with tiered storage before turning this on in evergreen. We don't want to turn evergreen red with lots of failures that we could have address by running additional testing ourselves.

Reply:
Sounds good, Don. Thanks!